fuzzer/src/main.rs

```diff
@@ -0,0 +1,223 @@
+#[macro_use]
+extern crate honggfuzz;
+
+use felt::Felt;
+use num_traits::Zero;
+use cairo_rs::vm::runners::cairo_runner::ExecutionResources;
+use starknet_rs::{
+    business_logic::{
+        execution::{
+            execution_entry_point::ExecutionEntryPoint,
+            objects::{CallInfo, CallType, TransactionExecutionContext},
+        },
+        fact_state::{
+            contract_state::ContractState, in_memory_state_reader::InMemoryStateReader,
+            state::ExecutionResourcesManager,
+        },
+        state::cached_state::CachedState,
+    },
+    definitions::{constants::TRANSACTION_VERSION, general_config::StarknetGeneralConfig},
+    services::api::contract_class::{ContractClass, EntryPointType},
+    utils::{calculate_sn_keccak, Address},
+};
+
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
+};
+
+use std::fs;
+use std::process::Command;
+
+
+fn main() {
+    println!("Starting fuzzer");
+    let mut iteration = 0;
+    fs::create_dir("cairo_programs");
```

Comment:
This returns a `Result<()>`, which should be checked.
```suggestion
    fs::create_dir("cairo_programs").expect("Failed to create cairo_programs/ directory");
```
Although looking later on, I think using a temporary directory (see [tempfile](https://crates.io/crates/tempfile)) would be fine too.

Reply:
i didin´t use the tempfile cause it was hard to mix with the way the files are created, but the result is checked now